examples/kinetic-electrons/wall+sheath-bc_boltzmann_loworder.toml

```
@@ -49,10 +49,10 @@ ionization_frequency = 2.0
 #electron_ionization_frequency = 2.0
 #ionization_energy = 1.0
 constant_ionization_rate = false
-nstep = 1000000
+nstep = 1000
 #nstep = 1
 dt = 1.0e-5
-nwrite = 10000
+nwrite = 100
```

Lets revert this change too
```suggestion
nwrite = 10000
```
and (if you see this comment in time) a nice thing to do with similar/related suggestions is to group them together with the 'Add suggestion to batch' button, because that makes a single commit rather than several (which makes the history a little bit tidier).